README.md

```diff
@@ -340,6 +340,104 @@ To view the metrics and traces port-forward the services:
 
 Then navigate to http://localhost:9090 for metrics and http://localhost:16686 for traces.
 
+## Datadog
+
+Keramik can also be configured to send metrics and telemetry data to datadog.
+
+We will following these steps.
+
+0. Setup Keramik network
+1. Install the operator
+2. Configure the operator
+3. Configure secrets for the operator
+4. Configure datadog within a Keramik network.
+
+The datadog operator needs to be installed into an existing Keramik network.
+Create Keramik network using `small.yaml` from this README.
+
+    kubectl apply -f small.yaml
+
+Install the datadog k8s operator into the network.
+This requires installing `helm`, there doesn't seem to be any other way to install the operator without first installing helm.
+However once the datadog operator is installed helm is no longer needed.
+
+    helm repo add datadog https://helm.datadoghq.com
+    helm install -n keramik-small my-datadog-operator datadog/datadog-operator
+
+Configure the datadog operator:
```

Comment:
What do we mean by configure? Do we add this file, or does helm add it for us and we need to adjust it?

Reply:
This config file adds an instance of the operator to the namespace. The `helm` commands above just install it (i.e. add the CRDs etc) but do not start the operator running.